.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
We dont need husky at all for this, we already have a pre commit hook right? just modify that

Reply:
Remove the dependency on husky